code/datums/uplink/gear loadout.dm

```diff
@@ -54,17 +54,17 @@
 /datum/uplink_item/item/gear_loadout/dominia_single
 	name = "Empire of Dominia Assets (Single)"
 	path = /obj/structure/closet/crate/secure/gear_loadout/dominia/single
-	item_cost = 15
+	item_cost = 5
 
 /datum/uplink_item/item/gear_loadout/cowboys
 	name = "Frontier Cowboys (Group)"
 	path = /obj/structure/closet/crate/secure/gear_loadout/ram_ranch
-	item_cost = 50
+	item_cost = 15
```

Comment:
See above

Reply:
Cost set to 35. Might consider buffing the frontier cowboys set to have more goodies even though it is a meme loadout choice.